dockerfiles/cosmos-exporter/v0.3.0/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM alpine:3.16
+EXPOSE 9300
+USER root
+
+RUN wget -q https://github.com/solarlabsteam/cosmos-exporter/releases/download/v0.3.0/cosmos-exporter_0.3.0_Linux_x86_64.tar.gz &&\
+    tar xvfz cosmos-exporter* -C /usr/bin
+
+ENTRYPOINT [ "cosmos-exporter" ]
+CMD [ "--denom", "uknow" ]
```

Comment:
As it will be a public image, it would be nice if it works with other blockchain but we can do this in an other PR. Maybe pass a config file.